### DISCUSSION TITLE
Duda correciones de tps

### DISCUSSION THREAD

**Discussion by NicoHermosilla** [INGCOM-UNRN-P1/cursada-2023#41] [Preguntas y respuestas] [answered]
Buenas queria preguntar que pasa si yo hoy entrego el tp 1 y tiene correciones para hacer, ¿tengo tiempo hasta fecha limite para corregirlo o el tiempo de correcion es diferente al de la fecha limite?

**Comment by martinvilu** [comment, marked as answer]
La fecha de entrega sigue siendo la misma, podemos dar algún margen, que veremos por entrega.
